unyt/unit_symbols.py

```diff
@@ -179,6 +179,12 @@
 Pa = pascal = quan(1.0, "Pa")
 #: sievert
 Sv = sievert = quan(1.0, "Sv")
+#: candela
+cd = Candela = candela = quan(1.0, "cd")
+#: lumen
+lm = Lumen = lumen = quan(1.0, "lm")
+#: lux
+lx = Lux = lux = quan(1.0, 'lx')
```

Comment:
Add [Lambert](https://en.wikipedia.org/wiki/Lambert_(unit)) here. Maybe some of the other luminance units as well?